packages/exo/src/exo-tools.js

```diff
@@ -32,6 +32,13 @@ const { defineProperties, fromEntries } = Object;
  */
 const MinMethodGuard = M.call().rest(M.any()).returns(M.any());
 
+/**
+ * @param {Passable[]} args
+ * @param {MethodGuardPayload} methodGuardPayload
+ * @param {string | undefined} label
```

Comment:
consider JSDoc convention:
```suggestion
 * @param {string} [label]
```

Reply:
I had not intended to make it optional, but rather, merely allow its value to be `undefined`. It isn't exported and its two callers in this file provide the `label` argument explicitly.

However, I don't mind making it optional, so done.